src/globals.c

```diff
@@ -127,35 +127,31 @@ static int set_url(char **global, char *url, const char *path)
 		} else {
 			/* no option passed; use the default value */
 			ret = set_default_value(global, path);
```

Comment:
Can you rename this function to `set_default_value_from_path` instead, to better match the new semantics?